src/Orchard.Web/Modules/Orchard.Workflows/Controllers/AdminController.cs

```diff
@@ -303,9 +303,27 @@ IFormManager formManager
                 });
             }
 
+            if (clearWorkflows) 
+                workflowDefinitionRecord.WorkflowRecords.Clear();

```

Comment:
Is it really deleting the workflow records or just removing them from the list instead?